scripts/upset_plot_ranked.py

```diff
@@ -4,6 +4,7 @@
 import os
 import pandas
 import pickle as pkl
+# # TODO: Do we need to add upsetplot to our environment yaml?
```

Comment:
yes! first I think we should figure out the best way to save the env: https://github.com/choderalab/covid-moonshot-ml/issues/41